src/code/audioMgr.c

```diff
@@ -56,7 +56,7 @@ void AudioMgr_ThreadEntry(void* arg0) {
     osSyncPrintf("オーディオマネージャスレッド実行開始\n"); // "Start running audio manager thread"
     Audio_Init();
     AudioLoad_SetDmaHandler(DmaMgr_DmaHandler);
-    Audio_InitSound();
+    Audio_InitSfx();
```

Comment:
Is this really a "sfx" thing? It looks like it does more

Reply:
It looks like 4 of the 6 are for sfxs, but yeah it has some general sequence stuff thrown in there. So I'll keep it as Sound